docs/registry/_index.md

```diff
@@ -96,87 +102,28 @@ Once you have added and configured the module you would like to use in the Viam
 
 `viam-server` manages the [dependencies](/internals/rdk/#dependency-management), [start-up](/internals/rdk/#start-up), [reconfiguration](/internals/rdk/#reconfiguration), [data management](/services/data/capture/#configure-data-capture-for-individual-components), and [shutdown](/internals/rdk/#shutdown) behavior of your modular resource.
 
-### Tutorials using modules
-
-{{< cards >}}
-{{% card link="/tutorials/projects/make-a-plant-watering-robot/" %}}
-{{% card link="/tutorials/projects/integrating-viam-with-openai/" %}}
-{{< /cards >}}
-
 ## Create your own modules
 
-If none of the existing modular resources in the Viam registry support your use case, you can create your own modules with your own modular resources:
+If none of the existing modular resources in the Viam registry support your use case, you can create your own modules to provide your own modular resources:
 
 - **Implement a custom component**: Write a driver for an unsupported {{< glossary_tooltip term_id="component" text="component" >}} by implementing the corresponding component API.
 
 - **Implement a custom service**: Implement your own algorithm or {{< glossary_tooltip term_id="model" text="model" >}} against a corresponding service API or use custom algorithms or data models when working with services such as {{< glossary_tooltip term_id="slam" text="SLAM" >}}, vision, or motion planning.
 
 You can write modules in a variety of programming languages, such as, Go, Python, C++, Rust, while implementing the same [APIs](/appendix/apis/).
+
 To create a new module:
 
-1. [Create a module](/registry/create/) with one or more modular resources by implementing all methods for the component's or service's standardized API.
-1. [Upload the module to the Viam registry](/registry/upload/) to make it available for deployment to machines or add it as a [local module](/registry/configure/#local-modules).
+1. [Create a module](/use-cases/create-module/) with one or more modular resources by implementing all methods for the component's or service's standardized API.
+1. [Upload the module to the Viam registry](/use-cases/create-module//#upload-your-module-to-the-modular-resource-registry) to make it available for deployment to machines or add it as a [local module](/registry/configure/#local-modules).
    You can upload _private_ modules for your {{< glossary_tooltip term_id="organization" text="organization" >}} or _public_ modules.
 1. Once you have uploaded your module to the registry, [deploy and configure the module](/registry/configure/) from the Viam app.
    Then, you can test your added resource using the [**CONTROL** tab](/fleet/control/) and [program](/build/program/) it with Viam's Go or Python SDKs.
 
-## Naming your model: namespace:repo-name:name
```

Comment:
I think we also need this info in reference style. Maybe in a model subpage? maybe copy what's here into a ticket. It doesn't need to be done now.

Reply:
Hmm this is part of the larger convo about reference and how-tos. I will create a ticket so we don't lose this thread but my personal take right now is that people should just look at this here; the info is all here, in one place, and we can link to it.